services/audit-service/src/main/java/datawave/microservice/audit/replay/util/ConcurrentMapCacheInspector.java

```diff
@@ -0,0 +1,61 @@
+package datawave.microservice.audit.replay.util;
```

Comment:
hazelcast-common now includes a ConcurrentMapCacheInspector, so you might be able to drop this class.